visualisation/backend/influx_db8.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef SERENITY_INFLUX_DB8_HPP
+#define SERENITY_INFLUX_DB8_HPP
+
+#include <string>
+
+#include "utils.hpp"
+
+#include "visualisation_backend.hpp"
+
+namespace mesos {
+namespace serenity {
+
+constexpr char DEF_INFLUX_DB_ADDRESS[] = "localhost";
```

Comment:
Can we inline these in the constructor? Or do you use these constants other places?